.github/workflows/androidBump.yml

```diff
@@ -0,0 +1,40 @@
+name: Android Rollout Bumper
+
+on:
+  schedule:
+    # Runs at midnight every day
+    - cron: '0 0 * * *'
+
+jobs:
+  android_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: ./.github/actions/composite/setupNode
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.190.0
+        with:
+          bundler-cache: true
+
+      - name: Decrypt json Google Play credentials
+        run: gpg --batch --yes --decrypt --passphrase="${{ secrets.LARGE_SECRET_PASSPHRASE }}" --output android-fastlane-json-key.json android-fastlane-json-key.json.gpg
+        working-directory: android/app
+
+      - name: Generate version
+        id: checkAndroidStatus
+        uses: ./.github/actions/javascript/checkAndroidStatus
+        with:
+          GOOGLE_KEY_FILE: android/app/android-fastlane-json-key.json
+          PACKAGE_NAME: org.me.mobiexpensifyg
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update Rollout Percentage with Fastlane
+        if: ${{ steps.checkAndroidStatus.outputs.HALTED == 'false' }}
```

Comment:
Should we also return early if rollout is at `1`?

Reply:
+1

Reply:
Ah yes, we should

Reply:
I guess actually, we need to check if the current rollout is 1, not if the next rollout will be 1.